src/uu/od/src/od.rs

```diff
@@ -286,26 +224,252 @@ impl OdOptions {
+        )
+        .arg(
+            Arg::with_name(options::STRINGS)
+                .short("S")
+                .long(options::STRINGS)
+                .help(
+                    "output strings of at least BYTES graphic chars. 3 is assumed when \
+                     BYTES is not specified.",
+                )
+                .takes_value(true)
+                .default_value("3")
+                .value_name("BYTES"),
+        )
+        .arg(
+            Arg::with_name("a")
+                .short("a")
+                .help("named characters, ignoring high-order bit")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("b")
+                .short("b")
+                .help("octal bytes")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("c")
+                .short("c")
+                .help("ASCII characters or backslash escapes")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("d")
+                .short("d")
+                .help("unsigned decimal 2-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("D")
+                .short("D")
+                .help("unsigned decimal 4-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("o")
+                .short("o")
+                .help("octal 2-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("I")
+                .short("I")
+                .help("decimal 8-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("L")
+                .short("L")
+                .help("decimal 8-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("i")
+                .short("i")
+                .help("decimal 4-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("l")
+                .short("l")
+                .help("decimal 8-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("x")
+                .short("x")
+                .help("hexadecimal 2-byte units")
+                .multiple(true)
+                .takes_value(false),
+        )
+        .arg(
+            Arg::with_name("h")
+                .short("h")
+                .help("hexadecimal 2-byte units")
+                .multiple(true)
+                .takes_value(false),
```

Comment:
I think takes_value is the default. please remove them

btw, you set .multiple(true), is that necessary ?

Reply:
Done with the `takes_value`

Yes, it is necessary since `od` might receive the same flag multiple times, if i remove it clap will complain